app/Models/Configs.php

```diff
@@ -35,6 +35,7 @@ class Configs extends Model
 	use UseFixedQueryBuilder;
 
 	protected const INT = 'int';
+	protected const INTM = 'intm';
```

Comment:
What is an "intm"?

Reply:
I found out the hard way that someone before me came to the idea that an 'int' is by lychee definition is a positive integer number. Since I need a negative value in my configuration, I needed a definition of an integer which supports  negative numbers. I think that "intm" fits perfectly in relation to the definition of 'int' as a positive number. You might come up with something better. The best would be to define 'int' as full range integer and 'positive_int' as the positive only version.

Reply:
> I found out the hard way that someone before me came to the idea that an 'int' is by lychee definition is a positive integer number. Since I need a negative value in my configuration, I needed a definition of an integer which supports negative numbers. I think that "intm" fits perfectly in relation to the definition of 'int' as a positive number. You might come up with something better. The best would be to define 'int' as full range integer and 'positive_int' as the positive only version.

Sorryyyyyyy

Reply:
> I found out the hard way that someone before me came to the idea that an 'int' is by lychee definition is a positive integer number.

Well, it seems that back then only positive integers were required. I believe, @ildyria just apologized for it. However, I believe that this has been OK in the sense that those things must always be evaluated on the background at that time.

> I think that "intm" fits perfectly in relation to the definition of 'int' as a positive number. You might come up with something better.

Sorry, but I disagree. "intm" ist just bad. I have just realized that it is supposed to mean "int minus", but I don't want to guess when I am reading code. If you need signed integers, it is now the time to fix that. So, please rename the first constant into `UNSIGNED_INTEGER` (or for my sake `UNSIGNED_INT` if you want it short) and `SIGNED_INTEGER`. That makes things clear and future developers won't need to guess.

And no, it is not up to us, to come up with something better. It is your PR.

Reply:
I changed the configuration entry to 'signed_int', someone else might change 'int' to 'unsigned_int' if he likes to do it.